app.py

```diff
@@ -0,0 +1,97 @@
+import panel as pn
+import hvplot.pandas
+
+# Load Data
+from data import per_dataset_df, all_framework_df
+from widgets import create_selectwidget, create_togglewidget, create_numberwidget
+from plots import create_hvplot,create_table
+
+#clean up framework names
+original_framework_names = all_framework_df['framework']
```

Comment:
Instead of using string `"framework"`, better create an function and take in an argument for `framework_names`, so we can easily update it if it changes